### PR TITLE
Adding Hopper to list

### PR DESCRIPTION
**Pull request by ngutierrez31**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [ ] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/hopper.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Hopper is a company developing a mobile app for booking flights and hotels. It predicts and analyzes airfare and accommodation helping users to plan, book, and pay trips.
+
+## Company size
+
+Hopper has 437 employees around the globe.
+
+## Remote status
+
+Some employees work full-time from our offices and some work
+remotely. There are a variety of hybrid and fully remote roles at Hopper.
```
Comment (dougaitken) [inline company-profiles/hopper.md:14]: There is an odd line break here

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/hopper.md (hunk 1 above)
Comment (ngutierrez31) [reply company-profiles/hopper.md:14]: @dougaitken I fixed the line break.

Comment (dougaitken) [reply company-profiles/hopper.md:14]: It is still there. The line break between `some work` and `remotely`

**Comment by ngutierrez31** [comment]
@dougaitken my mistake. I miss understood the comment originally. I think it's all set now :)

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Can you take a look at the merge conflict, please @ngutierrez31?

**Comment by ngutierrez31** [comment]
@dougaitken should be all set.

**Review by dougaitken — APPROVED** [review]

**Pull request merged**